toolbox/GEE_Connector.pyt

```diff
@@ -19,7 +19,7 @@ import json
 import requests
 from google.cloud import storage
 import arcgee
-
+from pathlib import Path
```

Comment:
Sort imports. pathlib is  builtin and should be in the intial group. And use `import pathlib`

```python
import json
import os
import pathlib

import arcgee
import arcpy
import ee
from google.cloud import storage
import requests
```